src/FloatingActionButton/FloatingActionButton.js

```diff
@@ -270,7 +270,8 @@ class FloatingActionButton extends Component {
       style: Object.assign({},
         styles.icon,
         mini && styles.iconWhenMini,
-        iconStyle),
+        iconStyle,
+        this.props.children.props.style),
```

Comment:
Should it be the opposite order? Where the parent have more precedence over the child.
```jsx
         this.props.children.props.style,
         styles.icon,
         mini && styles.iconWhenMini,
         iconStyle,
),
```